controller/search.go

```diff
@@ -140,53 +143,127 @@ func (c *SearchController) WorkitemsCSV(ctx *app.WorkitemsCSVSearchContext) erro
 	if ctx.FilterExpression == nil {
 		return goa.ErrBadRequest(fmt.Sprintf("bad parameter error exporting work items as CSV: param 'filter[expression]' missing"))
 	}
-	var result []workitem.WorkItem
+	if ctx.PageLimit != nil && *(ctx.PageLimit) > 1000 {
+		// the window size is too large
+		return goa.ErrBadRequest("maximum limit value is 1000")
```

Comment:
Instead of returning an error, how about changing the limit to maximum possible value?

Reply:
I don't thin this is a good idea as we possibly need to deal with automated clients like scipts etc. that may expect the given number of returned items. For a user endpoint, I'd agree though.

Reply:
I agree with @michaelkleinhenz